### PR TITLE
chore: add job to push changes

### PR DESCRIPTION
**Pull request by tac0turtle**

### CODE REVIEW

File: versioned_docs/version-0.50/build/modules/authz/README.md [hunk 1]
```diff
@@ -1,355 +1,19 @@
 ---
-sidebar_position: 1
+sidebar_position: 0
 ---
 
-# `x/authz`
+# Tools
```
Comment (tac0turtle) [inline versioned_docs/version-0.50/build/modules/authz/README.md:5]: i think theres a bug in the script

**Comment by samricotta** [comment]
What exactly is broken? I think it might be the folders within the branches of the other repo which arent aligned exactly. It was working for me locally but ill have another check again

**Comment by tac0turtle** [comment]
it seems like everything is being replaced with tools readme. Please have a look through the changelog. The script should not be overwriting x/authz docs with tools.

File: versioned_docs/version-0.50/build/modules/circuit/README.md [hunk 2]
```diff
@@ -1,151 +1,19 @@
-# `x/circuit`
+---
+sidebar_position: 0
+---
 
-## Concepts
+# Tools
```
Comment (tac0turtle) [inline versioned_docs/version-0.50/build/modules/circuit/README.md:5]: its all getting set to tools

**Pull request closed without merging**